src/02_datatypes.py

```diff
@@ -13,9 +13,12 @@
 
 # Write a print statement that combines x + y into the integer value 12
 
-# YOUR CODE HERE
+total = int(x) + int(y)
```

Comment:
this is perfect! 
you could save a few characters by not forcing x to be an int here, since it is already of type int. 
it could be better to force both types though, for readability, so this is perfectly fine. :D great job!